OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -120,10 +120,14 @@ void CaptureClient::Capture(int32_t process_id, std::string process_name,
   while (!force_stop_ && reader_writer_->Read(&response)) {
     event_processor.ProcessEvents(response.capture_events());
   }
-  LOG("Finished reading from Capture's gRPC stream: all capture data has been "
-      "received");
-
-  capture_listener_->OnCaptureComplete();
+  if (force_stop_) {
+    capture_listener_->OnCaptureFailed(
+        ErrorMessage("WritesDone on Capture's gRPC stream failed: unable to finish the "
+                     "capture in orderly manner, initiating emergency stop"));
```

Comment:
In this case the emergency stop is just happened. s/initiating emergency stop/capture halted/?

Reply:
Done.